models/redhat/redhat.go

```diff
@@ -101,7 +101,7 @@ func collectRedHatPacks(cri Criteria) []models.Package {
 	ps := walkRedHat(cri, []models.Package{}, "")
 	pkgs := map[string]models.Package{}
 	for _, p := range ps {
-		pkgs[p.Name] = p
+		pkgs[p.Name+p.Version] = p
```

Comment:
It would be better to add a modularitylabel than a version.
```suggestion
		n := p.Name
		if p.ModularityLabel != "" {
			n = fmt.Sprintf("%s::%s", p.ModularityLabel, p.Name)
		}
		pkgs[n] = p
```